scripts/pharmgkb/drug_gene_relations/pharm.py

```diff
@@ -0,0 +1,644 @@
+
+    This is used to format many columns from the drugs and genes dataframes like
+    PubChem Compound Identifiers or NCBI Gene ID.
+
+    Args:
+        text_list: A single string representing a comma separated list. Some of
+            the items are enlcosed by double quotes and some are not.
+
+    Returns:
+        A string that is mcf property text values list enclosed by double quotes
+        and comma separated.
+
+    Example:
+        input: ('test1,
+            "Carbanilic acid, M,N-dimethylthio-, O-2-naphthyl ester", "test2"')
+        return: ('"test1",
+             "Carbanilic acid, M,N-dimethylthio-, O-2-naphthyl ester", "test2"')
+    """
+    if not text_list:
+        return ''
+    formatted_str = ''
+    joining = ''
+    for prop_value in text_list.split(','):
+        if prop_value.count('"') == 0 and joining:
+            joining += prop_value + ','
+        elif prop_value.count('"') == 0 or prop_value.count('"') == 2:
+            formatted_str += '"' + prop_value.replace('"', '').strip() + '",'
+        elif prop_value.count('"') == 1 and joining:
+            formatted_str += joining.strip() + prop_value.replace('"',
+                                                                  '') + '",'
+            joining = ''
+        elif prop_value.count('"') == 1:
+            joining = prop_value + ','
+        else:
+            print('unexpected format: ' + text_list)
+    return formatted_str.strip(',')
+
+
+def format_semicolon_list(semi_list):
+    """Formats a string representing a semi colon separated list into MCF
+    property values list format.
+
+    This is used to format 'PMIDs' in relationships.tsv.
+
+    Args:
+        semi_list: a string representing a semi colon separated list
+
+    Returns:
+        A string that is mcf property text values list enclosed by double quotes
+        and comma separated.
+    """
+    if not semi_list:
+        return ''
+    formatted_str = ''
+    for prop_value in semi_list.split(';'):
+        formatted_str += '"' + prop_value + '",'
+    return formatted_str.strip(',')
+
+
+def format_bool(text, true_val):
+    """Checks to see if given value matches the value that should return True.
+
+    This is used to format the boolean values from 'PK' and 'PD' columns from
+    relationships.tsv. For example, the values in 'PK' column are 'PK' if the
+    relationship is pharmacokinetic, thus this function should return true only
+    if the value is equal to 'PK'. We cannot conclude that a relationship is not
+    pharmacokinetic by lack of 'PK' value, thus 'True' and empty are the only
+    possible values.
+
+    Args:
+        text: the raw value that needs to be checked
+        true_val: the value that 'text' should be to return true
+
+    Returns:
+        'True' (as a string) if 'text' matches 'true_val' and an empty string
+        otherwise.
+    """
+    if text == true_val:
+        return 'True'
+    return ''
```

Comment:
Why is this an empty string instead of False?

Reply:
added info in the method header to address this